charts/mattermost-enterprise-edition/templates/deployment-mattermost-app.yaml

```diff
@@ -37,6 +37,30 @@ spec:
         {{- .Values.mattermostApp.extraPodAnnotations | toYaml | nindent 8 }}
         {{- end }}
     spec:
+      {{if .Values.mattermostApp.antiAffinity.enabled}}
+      {{- if eq .Values.mattermostApp.antiAffinity.mode "hard" }}
```

Comment:
instead of having this the user can define his own affinity rules, so you just add the affinity object to be configured

Reply:
What do you suggest having instead, putting stale memory/cpu values?
or the soft hard ones

Reply:
the user defines the whole affinity
in the template you have just: 
```
    {{- if .Values.server.affinity }}
      affinity:
{{ toYaml .Values.server.affinity | indent 8 }}
    {{- end }}
```

and in the values: 
```
  affinity: {}
```

Reply:
how about giving the 2 options?
putting soft/hard and giving the user putting hes own custom affinity
Values.server.customaffinity - something like that